src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -196,6 +197,7 @@ public partial struct JsonWriterState
         public bool TryGetDateTimeOffset(out DateTimeOffset value) { throw null; }
         public bool TryGetDecimal(out decimal value) { throw null; }
         public bool TryGetDouble(out double value) { throw null; }
+        public bool TryGetGuid(out Guid value) { throw null; }
```

Comment:
`System.Guid`

Can you regenerate the ref assembly by running msbuild /t:GenerateReferenceSource from the ref directory?

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/updating-ref-source.md